contracts/protocore/GenesisOriginProtocore.sol

```diff
@@ -24,18 +24,36 @@ contract GenesisOriginProtocore {
 
     /* Storage */
 
-    /** Block hash of source checkpoint */
+    /** Metachain id of the origin chain. */
+    bytes32 public genesisOriginMetachainId;
+
+    /** Epoch length. */
+    uint256 public genesisEpochLength;
+
+    /** Initial metablock height. */
+    uint256 public genesisMetablockHeight;
+
+    /** Initial dynasty number */
+    uint256 public genesisDynasty;
```

Comment:
Dynasty should not be defined in *Protocore contracts. It should be part of Coconsensus contract.

Reply:
Removed `genesisDynasty` from origin protocore.